libsolidity/interface/ABI.cpp

```diff
@@ -121,6 +121,26 @@ Json::Value ABI::generate(ContractDefinition const& _contractDef)
 		abi.emplace(std::move(event));
 	}
 
+	// TODO should use "used errors" once implemented,
+	// but should also include all errors defined in the contract.
+	for (ErrorDefinition const* error: _contractDef.errors())
+	{
+		Json::Value event;
+		event["type"] = "error";
+		event["name"] = error->name();
+		Json::Value params(Json::arrayValue);
```

Comment:
```suggestion
		event["params" ] = Json::arrayValue;
```
Than later when doing `params.append` you can `event["params" ].append`

But not necessary to change this